src/manifest.c

```diff
@@ -675,19 +675,21 @@ char *file_type_to_string(struct file *file)
 	return type;
 }
 
+/* Calculate the contentsize for the manifest based on file sizes.
+ * This should calculate the files uniquely included in this manifest, but none
+ * of its submanifests, which will allow calculation of all bundles on a system
```

Comment:
instead of " ... calculation of all bundles ..." the above should say "... calculation of sizes of all bundles ...